mkdocs.yml

```diff
@@ -8,6 +8,30 @@ theme:
   language: 'en'
   features:
     # - navigation.top
+
+  # Respect the users default settings and add adjustable light/darkmode toggle.
+  # taken from https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#system-preference
+  palette:
+
+    # TODO: enable this after we upgrade mkdocs-material above "^9.5"
+    # Palette toggle for automatic mode
+    # - media: "(prefers-color-scheme)"
+    #   toggle:
+    #     icon: material/brightness-auto
+    #     name: Switch to light mode
+    # Palette toggle for light mode
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+    # Palette toggle for dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      toggle:
+        icon: material/brightness-4
+        name: Switch to system preference
```

Comment:
The name seems swapped between this and the automatic scheme.

Reply:
fixed, this was some bad copypasta.